haiku-apps/projectconceptor/projectconceptor-1.0.0.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A Haiku diagramming application like Microsoft Visio"
+DESCRIPTION="ProjectConceptor is a modular application for Haiku and BeOS \
+designed to view, edit and process information which can be represented by a \
+graph structure."
+APP="ProjectConceptor"
+HOMEPAGE="https://github.com/Paradoxianer/$APP"
+COPYRIGHT="2017 Matthias “Paradoxer” Lindner
```

Comment:
```diff
-COPYRIGHT="2017 Matthias “Paradoxer” Lindner
+COPYRIGHT="2017 Matthias \“Paradoxer\” Lindner
```
(I guess you want to see the double quotes :)

Reply:
@owenca, I saw the recent changes in the upstream repo, and this is great!

The latest stable release is 0.0.7, so I was wondering if jumping from 0.0.7 to 1.0.0 was not too "big" a jump, since the recent changes were mainly to make the app build on Haiku. Wouldn't a smaller jump, like 0.0.7 to 0.1.0 better reflect the recent changes?

Hmm, I saw that the project has an rsrc:
https://github.com/Paradoxianer/ProjectConceptor/blob/master/src/app/ProjectConceptor.rsrc

Would you like to convert it to an rdef? That would be terrific :)

Reply:
I believe I had tried to convert the rsrc file to rdef but `rc` couldn't decompile it. It also happened to some of the other apps I worked. Will double check though.

Reply:
Oops! It turns out there are a few more rsrc files. The ProjectConceptor.rsrc can be decompiled. Will try the others later and make a new PR to the upstream repo. We can either merge this one now or wait for the rdef files got merged upstream.

Reply:
Cool!
For the merge, we can do whatever you wish. If you prefer to have this PR merged before you convert the rsrc files, that's perfectly OK.
But since there is no urgency at all, it is also perfectly OK to keep this PR open, and wait for your future upstream PRs to get merged.
It's up to you.

Reply:
Let’s wait then.

Reply:
Actually i think moving from 0.0.7 to 0.1.0 makes much more sense,  since its not even beta yet.. (still 3 bugs left .. since 5 years or so ;) ) i just dont have to much time left ot work on it  :) But i am happy to upstream any usefull change :) And i can make a release  for 0.1.0 if this helps :-D

Reply:
btw. thanks for the recepie... this is something i didnt achived also i tried some weeks .(1h or 2h i have left every week)

Reply:
I will study the code in the summer (if not sooner) and try to fix the bugs. :)

Reply:
I alread told scott.. that my higher priority was / would be the index_server from haiku... actually it only need a good settings implemention then we could have a good full text search and thinks like thumbnails generation, mp3 methadata to attribute conversion and so on... 
i dont say projectconceptor isnt worth it .. but it would help tenthousden times more to get the index_server running... I am out of haiku since 6 Mont now.. but i didnt worked on index_server for 1 Year (did mostly testing) but if  you decide to work on this... i could try to help ;-)...

Reply:
I'd love to work on index_server with you! I'm not sure if it can be done as a GCI task, although there are only a few days left. Otherwise, I will see if I have time during spring break.